modin/pandas/utils.py

```diff
@@ -380,5 +381,48 @@ def _doc_binary_op(operation, bin_op, left="Series", right="right", returns="Ser
     return doc_op
 
 
+class NumericOnly(IntEnum):  # noqa: PR01
```

Comment:
there's possibly more than one method that doesn't tolerate numeric categories, so decided to add this enum